doc/specs/vulkan/chapters/memory.txt

```diff
@@ -1721,6 +1721,7 @@ ifdef::VK_KHX_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances.
 endif::VK_KHX_device_group[]
+  * pname:ppData must: be a valid pointer to a pointer
```

Comment:
> autogenerated new version would be:

> > pname:ppData must: be a valid pointer to a valid pointer

> Both would be lies.
> As the manual words say, the outer pointer does have to be "valid" the inner does not.

Huh. From looking at your script changes I'd have expected it to be what you wrote for the explicit VU. If you've tested this and are sure, fine - though it'd be nicer for it to not do this weirdness automatically!